### PR TITLE
[UPDATE BONUS GUIDE] Update aliases bonus guide (minor fixes)

### PR DESCRIPTION
**Pull request by twofaktor**
#### What

I realized was wrong information in the download aliases command and I had the idea to add a new command to show the aliases list directly in the terminal

### Why

This can lead to significant improvements to the correct running of the Raspibolt nodes.

#### How

- Fixed command to overwrite
- Updated redaction accordingly to change
- Added additional command to display current aliases list directly in the terminal

#### Scope

- [ ] significant change to core configuration
- [X] independent bonus guide
- [ ] simple bug fix

### CODE REVIEW

File: guide/bonus/raspberry-pi/aliases.md [hunk 1]
```diff
@@ -40,69 +40,86 @@ The following list of aliases was derived from contributions by [RobClark56](htt
...
-  ```sh
-  $ cd /home/admin
-  $ wget https://raw.githubusercontent.com/raspibolt/raspibolt/master/resources/.bash_aliases
-  ```
+```sh
+$ cd /home/admin
+$ wget https://raw.githubusercontent.com/raspibolt/raspibolt/master/resources/.bash_aliases -O .bash_aliases
+```
+
+⚠️ This command will automatically overwrite the previous file you downloaded.
```
Comment (VajraOfIndra) [inline guide/bonus/raspberry-pi/aliases.md:50]: Maybe put this warning just before the command itself.

File: guide/bonus/raspberry-pi/aliases.md [hunk 2]
```diff
@@ -40,69 +40,86 @@ The following list of aliases was derived from contributions by [RobClark56](htt
...
 
 * Inspect the list of aliases to make sure it does not do bad things and modify it with your personal aliases if you want. Exit with Ctrl-X
 
-  ```sh
-  $ nano .bash_aliases --linenumbers
-  ```
+```sh
+$ nano .bash_aliases --linenumbers
+```
```
Comment (VajraOfIndra) [inline guide/bonus/raspberry-pi/aliases.md:56]: Why did you remove the 2 spaces? We use them as standard format throughout the guide..

I didn't add more comments but this applies to all code blocks below.

File: guide/bonus/raspberry-pi/aliases.md [hunk 3]
```diff
@@ -40,69 +40,86 @@ The following list of aliases was derived from contributions by [RobClark56](htt
...
-  ```
+```sh
+$ source /home/admin/.bashrc 
+```
 
 ---
 
 ## Aliases in action
 
+* You can show the list of aliases to run the one you want typing `alias`
```
Comment (VajraOfIndra) [inline guide/bonus/raspberry-pi/aliases.md:68]:
```suggestion
* Simply type "alias" to display all available aliases
```

Just an idea for a simpler phrasing?

File: guide/bonus/raspberry-pi/aliases.md [hunk 4]
```diff
@@ -40,69 +40,86 @@ The following list of aliases was derived from contributions by [RobClark56](htt
...
+
+```sh
+...
```
Comment (VajraOfIndra) [inline guide/bonus/raspberry-pi/aliases.md:71]:
```suggestion
[...]
```
I've been using square quotes for missing text, as is done in literature.. but maybe this is not used for code writing, not sure. No need to change but simply wanting to raise the question.

**Review by VajraOfIndra — COMMENTED** [review]
LGTM! See a few comments below.
You removed the 2 spaces in front of the code blocks, why? We use them throughout the guide.

File: guide/bonus/raspberry-pi/aliases.md (hunk 1 above)
Comment (twofaktor) [reply guide/bonus/raspberry-pi/aliases.md]: Fixed!

File: guide/bonus/raspberry-pi/aliases.md (hunk 2 above)
Comment (twofaktor) [reply guide/bonus/raspberry-pi/aliases.md]: Fixed all code blocks!

File: guide/bonus/raspberry-pi/aliases.md (hunk 3 above)
Comment (twofaktor) [reply guide/bonus/raspberry-pi/aliases.md]: Fixed!

File: guide/bonus/raspberry-pi/aliases.md (hunk 4 above)
Comment (twofaktor) [reply guide/bonus/raspberry-pi/aliases.md]: Fixed!

**Comment by twofaktor** [comment]
All fixed! Review again!

**Review by VajraOfIndra — APPROVED** [review]
LGTM, tACK

Let's go to master :+1:
